docker/README.md

```diff
@@ -106,8 +106,10 @@ You can specify the following environment variables when issuing `docker run` co
 | `AWS_REGION` | string |  | AWS Region when using `s3` media handler |
 | `AWS_S3_BUCKET` | string |  | Name of the AWS S3 bucket when using `s3` media handler |
 | `AWS_SECRET_ACCESS_KEY` | string |  | AWS [Secret Access Key](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) when using `s3` media handler |
+| `CLUSTER_SELF` | string |  | Node name if the server is running in a Tinode cluster |
```

Comment:
Maybe `CLUSTER_NODE_NAME` or `CLUSTER_THIS_NODE_NAME`as more descriptive?

Reply:
Actually ignore this comment. `CLUSTER_SELF` is OK as it mirrors the command line parameter.